python/strings_udf/strings_udf/tests/test_length.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.
+
+import pytest
+
+from .utils import run_udf_test
+
+
+@pytest.mark.parametrize("data", [["cudf", "rapids", "AI", "gpu", "2022"]])
+def test_string_udf_len(data):
+    # tests the `len` function in string udfs
+
+    def func(st):
+        return len(st)
+
+    run_udf_test(data, func, "int64")
```

Comment:
Can you add a test using `len` directly? Or is a `__code__` attribute required as noted in #11595?

```python
    run_udf_test(data, len, "int64")
```

Reply:
Right - we'd need some separate handling for this. Adding the test yields
```python
AttributeError: 'builtin_function_or_method' object has no attribute '__defaults__'
```

Reply:
I noted this case in #11595 for now. We should revisit later.